quickstart/exampledata/config/pipeline.yml

```diff
@@ -57,7 +58,7 @@ input:
   kafka:
     type: confluentkafka_input
     bootstrapservers:
-      - 172.21.0.5:9092
+      - 127.0.0.1:9092
```

Comment:
Can you double check these changes please. I think the IP `172.21.0.5` is necessary for the quickstart setup.

Reply:
reverted changes on pipeline.yml